build/zip_bundle.gni

```diff
@@ -42,6 +42,7 @@ template("zip_bundle") {
     script = "//flutter/build/zip.py"
     outputs = [ "$root_build_dir/zip_archives/${invoker.output}" ]
     sources = []
+    forward_variables_from(invoker, [ "visibility" ])
```

Comment:
optional nit: the forwards usually come first, and it looks like you could also forward `deps`.

Reply:
done!